client/src/modules/profiles/pages/userProfile.tsx

```diff
@@ -94,9 +110,57 @@ export const UserProfilePage = () => {
             loadingText={'Saving Profile Changes'}
             submitText={'Save Profile Changes'}
           />
-          <Button colorScheme={'red'} marginBlock={'2em'} onClick={clickDelete}>
-            Delete My Data
-          </Button>
+          <Flex gap="2em" marginBlock={'2em'}>
+            <Button
+              height={'2.8em'}
+              paddingBlock={'.65em'}
+              paddingInline={'.4em'}
+              colorScheme={'red'}
+              onClick={clickDelete}
+            >
+              Delete My Data
+            </Button>
+            <Button
+              fontWeight="600"
+              background={'gray.85'}
+              color={'gray.10'}
+              height={'100%'}
+              size={'lg'}
+              borderRadius={'5px'}
+              paddingBlock={'.65em'}
+              paddingInline={'.4em'}
+              _hover={{ color: 'gray.85', backgroundColor: 'gray.10' }}
+              onClick={() => getData()}
+              isDisabled={!!userDataDownload}
+            >
+              Request your data
+            </Button>
+            {loading ? (
+              <Spinner />
+            ) : (
+              <>
+                {userDataDownload && (
+                  <Flex alignItems={'center'} gap="2em">
+                    <Text>The data was fetched succesfully 🥳</Text>
```

Comment:
```suggestion
```
I think the link alone is enough.

Reply:
👍 I will remove the flex element too, then.